src/ipsis/course-merge/BaseCollectionEntity.js

```diff
@@ -30,7 +21,7 @@ export class BaseCollectionEntity extends Entity {
 	}
 
 	loadMorePageSize() {
-		const pageSize = this._pagingInfo()?.pageSize;
+		const pageSize = 20;
```

Comment:
We can't get this from `this._pagingInfo()?.pageSize` anymore since the size of each additional load is not going to be the same as the current amount of loaded elements we have.
Is there a place where this constant is stored, or should we create a new one?